emulator.c

```diff
@@ -20,8 +20,11 @@ execute_instruction(i8080 *cpu, uint8_t opcode)
       printf("DCR B");
       break;
     case 0x06: // NOLINT
-      printf("MVI B");
-      break;
+      {        // MVI B, mem8
```

Comment:
Is this correct? cpu_read_mem takes in a 16-bit address but you're only passing in 8-bits. I think MVI B is move the data from the next byte into register b. Not find what's stored at the memory address of the next byte.